packages/server/src/fhir/operations/utils/terminology.ts

```diff
@@ -87,7 +87,7 @@ export function selectCoding(systemId: string, ...code: string[]): SelectQuery {
     .column('display')
     .where('system', '=', systemId)
     .where('code', 'IN', code)
-    .where('isSynonym', '=', false);
```

Comment:
Does this mean we will eventually drop the `isSynonym` column?

Reply:
Yeah, that would be the end goal: to replace `isSynonym` with `synonymOf`